modules/internal/ChapelRange.chpl

```diff
@@ -757,10 +763,10 @@ module ChapelRange {
     return lenAsUint: t;
   }
 
-  /* Return true if the range has a first index, false otherwise.
-     Note that in the event that the range is stridable and at least
-     partially bounded, the return value will not (cannot) be a
-     `param`.
+  /* Returns ``true`` if the range has a first index, ``false``
+     otherwise.  Note that in the event that the range is stridable
+     and at least partially bounded, the return value will not
+     (cannot) be a `param`.
```

Comment:
I find this "(cannot)" to be slightly distracting from the message. It seems a bit unnecessary and would more likely cause someone to wonder "why not?" and detract from the message, rather than provide any actual value to a user. Either they understand why it can't be so or they don't, no reason to call it out in my opinion.